GetCanvasTODOList/dcoder_block.yml

```diff
@@ -0,0 +1,18 @@
+id: TODO_LIST
+fileName: index.js
```

Comment:
Follow instruction and build into single file and turn off auto install, and update here dist/index.js

Reply:
Read process in the Contributing.md, without build dist blocks are not accepted.
Auto install module blocks are generally slow compare to using a dist build.